.github/workflows/canary-integration-test.yml

```diff
@@ -792,6 +787,7 @@ jobs:
           cat tests/manifests/test-on-pvc-db.yaml >> tests/manifests/test-cluster-on-pvc-encrypted.yaml
           cat tests/manifests/test-on-pvc-wal.yaml >> tests/manifests/test-cluster-on-pvc-encrypted.yaml
           kubectl create -f tests/manifests/test-cluster-on-pvc-encrypted.yaml
+          kubectl patch -n rook-ceph cephcluster rook-ceph --type merge -p '{"spec":{"security":{"keyRotation":{"enabled": true, "schedule":"*/5 * * * *"}}}}'
```

Comment:
> How often will the key rotation be tested here? 5 times per minute?

once every 5 minutes. https://crontab.guru/#*/5_*_*_*_* 

the script function `verify_key_rotation` 
checks secret for change in key value every 20 seconds for 10 minutes. 
once it is changed, 
`test osd deployment removal and re-hydration` test will take care of check of opening encrypted device with new key.

Reply:
Can we change this to rotate every minute? Then the CI wait can be much faster. I see the CI is waiting 2.5 minutes for this step, which would be nice to reduce.

Reply:
If less than one minute is a problem, why not rotate every 2 min? :) I know other jobs are longer, but if there are other changes in the future to this job, then it could become the longest.